src/main/java/frc/robot/Constants.java

```diff
@@ -14,6 +14,17 @@ public static class DriveTrainConstants {
     public static final int kLeftRearPort = 2;
     public static final int kRightFrontPort = 3;
     public static final int kRightRearPort = 4;
+    
+    // Feed Forward constants
+    public static final double kS = 0.0;
+    public static final double kV = 0.0;
+
+    // PID constants
+    public static final double kP = 0.0;
+    public static final double kI = 0.0;
+    public static final double kD = 0.0;
+
+    public static final double kDistanceToTarget = 0.0;
```

Comment:
Your distance to target is usually not a constant, it's value that changes as the game progresses.